src/components/shared/StyledNumberFormat.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import NumberFormat from 'react-number-format';
+import { makeStyles, Theme } from '@material-ui/core';
+
+/**
+ * Component for number formatting and optional styling
+ *
+ * @param value - number value that is being displayed
+ * @param suffix - suffix for the formatted number value, use autocompletion of your IDE!
+ * IMPORTANT: don't use curly braces around the suffix string
+ *    WRONG: suffix={'&nbsp;€'}
+ *    CORRECT: suffix='&nbsp;€'
+ * @param size - optional parameter to set the font size, if left out inherits from parent
+ * @param fontWeight - optional parameter to set the font weight, if left out inherits from parent
+ * @param paintJob - optional parameter to color the value:
+ *    - true will color green for positive values and red for negative values
+ *    - if a hex-string is passed (e.g. '#fff') the text will be painted in that color
+ *    - default/false is to inherit from parent (sets color to undefined)
+ */
+type StyledNumberFormatProps = {
+  value: number;
+  suffix: '&nbsp;€' | '&nbsp;%';
```

Comment:
Are there good reasons to include the space in the property like this instead of just setting it when the suffix is passed to the `NumberFormat` component?